src/c++/perf_analyzer/inference_profiler.cc

```diff
@@ -363,6 +415,11 @@ Report(
     ReportServerSideStats(summary.server_stats, 1, parser);
   }
 
+  if (should_collect_metrics) {
+    std::cout << "  Server Prometheus Metrics: " << std::endl;
```

Comment:
Does Prometheus mean anything to the user? Or is that just what we are using internally? Same comment for line 46

Reply:
[Prometheus](https://prometheus.io) is a tool mentioned here:
* https://github.com/triton-inference-server/server/blob/main/docs/user_guide/metrics.md
* https://github.com/triton-inference-server/server/blob/main/docs/user_guide/perf_analyzer.md#server-side-prometheus-metrics

Reply:
The original idea to include "Prometheus" in the feature name here came from here:

https://github.com/triton-inference-server/server/pull/4871#discussion_r969654259